src/caching/Arweave/ArweaveClient.ts

```diff
@@ -230,9 +230,9 @@ export class ArweaveClient {
     if (balanceInFloat.includes("e")) {
       const [balance, exponent] = balanceInFloat.split("e");
       const resultingBN = ethers.BigNumber.from(balance).mul(toBN(10).pow(exponent.replace("+", "")));
-      return parseWinston(resultingBN.toString());
+      return BigNumber.from(resultingBN.toString());
```

Comment:
Fair call - does this work on integration tests?

Reply:
yeah its the same type signature it just removes the extra 1e12 multiplication